src/ui/components/FiatInputValue/FiatInputValue.tsx

```diff
@@ -0,0 +1,82 @@
+import BigNumber from 'bignumber.js';
+import React from 'react';
+import { formatCurrencyValue } from 'src/shared/units/formatCurrencyValue';
+import { formatPercent } from 'src/shared/units/formatPercent/formatPercent';
+import { UIText } from 'src/ui/ui-kit/UIText';
+import { isNumeric } from 'src/shared/isNumeric';
+import { useCurrency } from 'src/modules/currency/useCurrency';
+import { exceedsPriceImpactThreshold } from 'src/ui/pages/SwapForm/shared/price-impact';
+import type { Asset } from 'defi-sdk';
+
+export function FiatInputValue({
+  name,
+  primaryInput,
+  spendInput,
+  spendAsset,
+  receiveInput,
+  receiveAsset,
+}: {
+  name: 'spendInput' | 'receiveInput';
+  primaryInput: 'spend' | 'receive';
+  spendInput?: string;
+  spendAsset: Asset | null;
+  receiveInput?: string;
+  receiveAsset: Asset | null;
+}) {
+  const { currency } = useCurrency();
+
+  const asset = name === 'receiveInput' ? receiveAsset : spendAsset;
```

Comment:
May be it is the right time to deprecate receiveInput editing?
This feature is not well-supported and backend suggests not to use it if possible.

Reply:
Wasn't aware that it isn't well-supported... It works perfectly in a swap form. Did you mean it isn't well supported only for bridge? Or in swap as well?

Reply:
Yes, I don't allow editing receiveInput in bridge, but regarding removing support for editing receiveInput in SwapForm... maybe lets discuss separately?